web/src/components/molecules/Asset/Asset/AssetBody/Asset.tsx

```diff
@@ -17,13 +17,8 @@ import SideBarCard from "@reearth-cms/components/molecules/Asset/Asset/AssetBody
 import UnzipFileList from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/UnzipFileList";
 import ViewerNotSupported from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/viewerNotSupported";
 import ArchiveExtractionStatus from "@reearth-cms/components/molecules/Asset/AssetListTable/ArchiveExtractionStatus";
-import {
-  fileFormats,
-  imageFormats,
-  compressedFileFormats,
-} from "@reearth-cms/components/molecules/Common/Asset";
+import { ViewerType } from "@reearth-cms/components/organisms/Asset/Asset/hooks";
```

Comment:
Shouldn't have cyclical imports. You should define this type in the molecule, and then import it, with the component, to the organism

Reply:
Done!